### PR TITLE
feat(BUX-643): updated README

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-js-client/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-js-client/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [x] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

**Review by chris-4chain — APPROVED** [review]
This will be outdated after my re-factorization process (BUX-611).
So, for now It looks good 👍 , but we have to keep in mind to update it once again after my modifications.

File: README.md [hunk 1]
```diff
@@ -35,16 +35,18 @@ See more scripts in the [package.json](package.json) file or the [makefile](Make
...
+| accessKeyString | Hex private part of an access key               | string      |
+| accessKey       | bsv PrivateKey instance                         |             |
+| adminKey        | Admin Key string                                | "xprv....." |
+| debug           | Whether to turn debugging on                    | boolean     |
+| signRequest     | Whether to sign all requests sent to the server | boolean     |
+| xPriv           | bsv HDPrivateKey instance                       |             |
+| xPrivString     | HD Private key string                           | "xprv....." |
+| xPub            | bsv HD Public key instance                      |             |
+| xPubString      | HD Public key string                            | "xpub....." |
+| xPubID          | XPub Id string                                  | string      |
```
Comment (arkadiuszos4chain) [inline README.md:49]: Are you sure we are able to make a request with xPubID only

**Review by arkadiuszos4chain — CHANGES_REQUESTED** [review]
Please update __Usage__ paragraph:
* replace handcash domain with `http://example.com/`
* use xpriv from string instead of `generateKeys()`

File: README.md (hunk 1 above)
Comment (Nazarii-4chain) [reply README.md]: Updated

**Comment by Nazarii-4chain** [comment]
> Please update **Usage** paragraph:
> 
> * replace handcash domain with `http://example.com/`
> * use xpriv from string instead of `generateKeys()`

Updated

**Pull request closed without merging**